dotCMS/src/main/java/com/dotcms/util/TimeUtil.java

```diff
@@ -0,0 +1,24 @@
+package com.dotcms.util;
+
+import com.dotmarketing.util.Config;
+
+import java.util.function.Supplier;
+
+public class TimeUtil {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)